src/coreclr/tools/dotnet-pgo/Program.cs

```diff
@@ -457,13 +458,13 @@ private int InnerCompareMibcMain()
             var paths = Get(_command.InputFilesToCompare);
 
             // Command line parser should require exactly 2 files
-            Trace.Assert(paths.Length == 2);
-            string file1 = paths[0];
-            string file2 = paths[1];
+            Trace.Assert(paths.Count == 2);
+            string file1 = paths.ElementAt(0).Value;
+            string file2 = paths.ElementAt(1).Value;
```

Comment:
This can reorder base/diff now. `compare-mibc` is sort of internal utility, it should be completely fine to change it to not support wild cards.

Reply:
Makes sense. Comparing exactly two files does not warrant wildcard support. I have kept it `BuildPathDictionay` in `merge` command.

Reply:
Makes sense. Comparing exactly two files does not warrant wildcard support. I have kept `BuildPathDictionay` in `merge` command.